src/app/components/surveillance/complaints/complaints.jsx

```diff
@@ -22,7 +26,23 @@ import { complaint as complaintPropType } from '../../../shared/prop-types';
 
 const useStyles = makeStyles(() => ({
   container: {
-    maxHeight: '40vh',
+    maxHeight: '64vh',
+  },
+  deleteButton: {
+    backgroundColor: '#c44f65',
+    color: '#ffffff',
+    '&:hover': {
+      backgroundColor: '#853544',
+    },
+  },
+  iconSpacing: {
+    marginLeft: '4px',
+  },
+  statusIndicatorOpen: {
+    color: '#66926d',
+  },
+  statusIndicatorClosed: {
+    color: 'rgba(0, 0, 0, 0.87)',
```

Comment:
This makes it slightly transparent, right? is there enough contrast with the transparency to meet 508 guidelines?

Reply:
This passes 508 contrast color checker on https://websemantics.uk/tools/wcag-colour-contrast-checker/